docs/README.md

```diff
@@ -13,4 +13,4 @@ If you are a researcher, a (bio)informatician, a biomedical practitioner, or any
 
 ### Documentation for previous versions
 
-[MOLGENIS v1.21.x](link TODO)
+[MOLGENIS v1.21.x](https://molgenis.gitbooks.io/molgenis/content/v/1.21/)
```

Comment:
User experience is better
* clear on each documentation page what version of the documentation you're browsing
* ability to go back and forward between versions

Developer experience is better
* (once set up) is easier to maintain cause all versions show same list from single source of truth

I've put up pull request to use it, can you take a look?